juniper_official/Solutions/Traffic-Blackhole/trap-stats.rule

```diff
@@ -0,0 +1,70 @@
+/*
+	* REQ: REQ-222
+	* Rule to get Trap Statistics (Drop Counters) for Blackhole Detection.
+	* Each platform will have different trap names.
+*/
+healthbot {
+	topic hardware.fpc {
+		rule trap-stats {
+			keys [ fpc-name trap-name ];
+			synopsis "Forwarding Block Trap Names and Values";
+			description "Forwarding Block Trap Names and Values";
+			sensor sensor-trap-stats {
+				synopsis "Forwarding Block Trap Names and Values";
+				description "Forwarding Block Trap Names and Values";
+				open-config {
+					sensor-name /junos/system/linecard/packet/usage/;
+					frequency 60s;
+				}
+			}
+			field fpc-name {
```

Comment:
For keeping common names, can we rename this to "fpc"? Keys section also should be adjusted accordingly

Reply:
Changed to fpc